src/components/top/myft-disengaged-header-tooltip/main.js

```diff
@@ -0,0 +1,39 @@
+import Tooltip from 'o-tooltip';
+import myftClient from 'next-myft-client';
+
+const myftHeaderLogo = document.querySelector('[data-trackable="my-ft"]');
+
+module.exports = (banner, done) => {
+	if (!myftHeaderLogo) return done();
+	myftClient.init([{ relationship: 'followed', type: 'concept' }]);
+	return myftClient.getAll('followed', 'concept')
+		.then(followedConcepts => {
+			if (followedConcepts.length) {
+				const concepts = followedConcepts
+					.sort((a, b) => b.lastPublished - a.lastPublished)
+					.map(({name}) => `<span style="no-wrap">${name}</span>`).slice(0, 3);
```

Comment:
```suggestion
					.map(({name}) => `<span style="no-wrap">${name}</span>`)
					.slice(0, 3);
```

to keep the style consistent :-)

Reply:
done